flora/config.h.example

```diff
@@ -1,20 +1,31 @@
-#define FLORA_ADDR "11:22:33:44:55:66"
 
-#define MQTT_BASE "flora/"
-#define MQTT_DEV_BASE MQTT_BASE FLORA_ADDR "/"
+// array of different xiaomi flora MAC addresses
+char* FLORA_DEVICES[] = {
+    "C4:7C:8D:67:11:11", 
+    "C4:7C:8D:67:22:22", 
+    "C4:7C:8D:67:33:33"
+};
 
-#define MQTT_TEMPERATURE MQTT_DEV_BASE "temperature"
-#define MQTT_MOISTURE MQTT_DEV_BASE "moisture"
-#define MQTT_LIGHT MQTT_DEV_BASE "light"
-#define MQTT_CONDUCTIVITY MQTT_DEV_BASE "conductivity"
-#define MQTT_BATTERY MQTT_DEV_BASE "battery"
+// sleep between to runs in seconds
+#define SLEEP_DURATION 30 * 60
+// emergency hibernate countdown in seconds
+#define EMERGENCY_HIBERNATE 3 * 60 * 1000
```

Comment:
Comment says "seconds", but you specify milliseconds here. CHange define to 
```suggestion
#define EMERGENCY_HIBERNATE 3 * 60
```
and also change delay coding below.